install.py

```diff
@@ -0,0 +1,270 @@
+#!/bin/python3
+
+#     ____                  __        ____
+#    / __ \____ ___________/ /_____ _/ / /
+#   / /_/ / __ `/ ___/ ___/ __/ __ `/ / /
+#  / ____/ /_/ / /__(__  ) /_/ /_/ / / /
+# /_/	 \__,_/\___/____/\__/\__,_/_/_/
+#
+# Copyright (C) 2020-2021
+#
+# This file is part of Pacstall
+#
+# Pacstall is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, version 3 of the License
+#
+# Pacstall is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Pacstall. If not, see <https://www.gnu.org/licenses/>.
+
+import os
+from sys import exit
+from shutil import chown, which
+from socket import create_connection
+from requests import get
+from subprocess import Popen, PIPE
+from multiprocessing import cpu_count
+from concurrent.futures import ThreadPoolExecutor
```

Comment:
Import the libs when required

Reply:
Doesn't affect performance. And it's more readible